pkgs/build-support/buildenv/builder.pl

```diff
@@ -117,10 +117,22 @@ sub addPkg {
     }
 }
 
+# Read packages list.
+my $pkgs;
+
+if (exists $ENV{"pkgsPath"}) {
+  local $/ = undef;
+  open FILE, $ENV{"pkgsPath"};
+  binmode FILE;
+  $pkgs = <FILE>;
+  close FILE;
+} else {
+  $pkgs = $ENV{"pkgs"}
+}
```

Comment:
Ouch, I'll fix the indentation -- thanks for noticing!

`binmode` and `$/` were taken from http://www.perlmonks.org/?node_id=1952, I was learning while making the patch ^_^. I'll try removing them and see if this works...